docs/transpile/transpiler-stages.ipynb

```diff
@@ -0,0 +1,355 @@
+      "text/plain": [
+       "<Figure size 370.906x200.667 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qiskit.providers.fake_provider import FakeSherbrooke\n",
```

Comment:
Could you tweak the first print statement a bit to make the output a bit easier to read? Something like:

`print("native gates: " + backend.operation_names)`